app/templates/components/events/view/overview/event-tickets.hbs

```diff
@@ -37,6 +37,8 @@
           <td>{{ticket.quantity}}</td>
           <td>{{sub ticket.quantity ticket.orderStatistics.tickets.completed}}</td>
         </tr>
+      {{else}}
+        <h4 class="ui header">{{t 'There are no tickets for this event'}}</h4>
```

Comment:
The helper text shown should have the same div class for all the tables in the event overview page. Tha tag h4 shouldn't be there.
Use: (as used in the manage roles table)
 <div class="ui basic segment">

Reply:
Understood, doing so